prime-router/src/main/resources/metadata/fhir_transforms/common/observation-nj-obx-dt-reformate.yml

```
@@ -0,0 +1,8 @@
+# suppressNonNPI: true
+# suppressNonNPI: true
+elements:
+- name: observation-nj-obx-DT-reformate
```

Don't need to to make this specific to NJ. It can be used by anybody that needs it and there's a typo on reformat. Can you also update the filename?
```suggestion
- name: observation-obx-DT-reformat
```